cmd/get_logs.go

```diff
@@ -203,6 +217,24 @@ func (glc *getLogsCmd) run() (err error) {
 		}
 	}
 	log.Infof("Logs downloaded to %s", glc.outputDirectory)
+	if glc.storageContainerSASURL != "" {
+		if glc.cs.Properties.IsCustomCloudProfile() && glc.cs.Properties.CustomCloudProfile.IdentitySystem == "adfs" {
+			log.Warn("Failed uploading logs for custom cloud with identity system of adfs")
+			return nil
+		}
+		logFiles, err := ioutil.ReadDir(glc.outputDirectory)
+		if err != nil {
+			return errors.Wrapf(err, "reading output directory %s", glc.outputDirectory)
+		}
+		for _, logFile := range logFiles {
```

Comment:
This will try to upload all files in the output directory, even those not produced by `get-logs`